virt/release_notes/virt-4-17-release-notes.adoc

```diff
@@ -81,6 +81,8 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-17-web"]
 === Web console
 
+* xref:../../virt/virtual_machines/virt-edit-vms.adoc#virt-hot-plugging-memory_virt-edit-vms[Hot plugging memory] for VMs from the web console is now generally available.
```

Comment:
A consideration: On core OCP, the release notes would step out the link in a "For more information..." clause. This would help those with accessibility needs.

Reply:
I think this is a great idea going forward and will bring it up to our team to be more in line with core OCP. But that would require changing more than just this RN so it's out of scope for this PR.